modules/nf-core/lofreq/indelqual/tests/main.nf.test

```diff
@@ -0,0 +1,75 @@
+nextflow_process {
+
+    name "Test Process LOFREQ_INDELQUAL"
+    script "../main.nf"
+    process "LOFREQ_INDELQUAL"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "lofreq"
+    tag "lofreq/indelqual"
+
+    test("sarscov2 - bam") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/bam/test.paired_end.sorted.bam', checkIfExists: true)
+                    ]
+                input[1] = [
+                    [ id:'ref' ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta', checkIfExists: true)
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                {
+                    def bam = file(process.out.bam.get(0).get(1))
+                    assert bam.exists()
```

Comment:
If assert snapshot(process.out).match() does not work, then probably check if a certain line exists in the file. Then if for some reason that's not possible, fallback to the .exists() check.

Reply:
In the past, I have struggled to get the string comparison of file content working. But I agree and will give it another shot. 👍